charts/prometheus-node-exporter/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.20.1
+version: 4.20.2
```

```suggestion
version: 4.21.0
```

Please, raise the minor number with a new feature.